tests/app/main/test_suppliers.py

```diff
@@ -2282,3 +2282,82 @@ def test_existing_org_size_sets_current_selection(self, data_api_client, existin
             doc = html.fromstring(res.get_data(as_text=True))
             selected_value = doc.xpath('//input[@name="organisation_size" and @checked="checked"]/@value')
             assert selected_value == expected_selection, 'The organisation size has not pre-populated correctly.'
+
+
+@mock.patch("app.main.views.suppliers.data_api_client", autospec=True)
+class TestSupplierAddRegisteredCompanyName(BaseApplicationTest):
+    def test_add_registered_company_name_page_loads(self, data_api_client):
+        with self.app.test_client():
+            self.login()
+            data_api_client.get_supplier.return_value = get_supplier(registeredName=None)
+
+            res = self.client.get("/suppliers/registered-company-name/edit")
+            assert res.status_code == 200, 'The add registered company name page has not loaded correctly.'
```

Comment:
What's the `'The add...'` string doing here?

Reply:
It's a custom assertion message that gets printed out if the test fails, to make it clear to a developer what's happened (in theory)